gaia_tasks/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+"""Top-level package for gaia tasks."""
+
+__author__ = """Kitware Inc"""
+__email__ = 'kitware@kitware.com'
+__version__ = '0.0.0'
+
+
+from girder_worker import GirderWorkerPluginABC
+
+
+class GaiaTasks(GirderWorkerPluginABC):
+    def __init__(self, app, *args, **kwargs):
+        self.app = app
```

Comment:
This function isn't needed.

Reply:
It seems without the __init__, girder-worker won't be able to pick up the task.